tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -2384,6 +2384,55 @@ public function test_remove_insecure_properties_applies_safe_styles() {
 		$this->assertEqualSetsWithIndex( $expected, $actual );
 	}
 
+	/**
+	 * @ticket 57321
+	 *
+	 * @covers WP_Theme_JSON::remove_insecure_properties
```

Comment:
```php
	 * @covers WP_Theme_JSON::remove_insecure_properties
```

@costdev with this as the class level, does including the class name again cause a problem for coverage reports?

https://github.com/WordPress/wordpress-develop/blob/64c4621fa73522af735c35c23f5494853394531a/tests/phpunit/tests/theme/wpThemeJson.php#L13

Reply:
I don't believe so.

That said: `@covers WP_Theme_JSON` at the class level could just be changed to `@coversDefaultClass WP_Theme_JSON`.

All test methods within this can then just use `@covers ::method_name` instead of `@covers WP_Theme_JSON::method_name`.